helm/etcd-backup-chart/templates/rbac.yaml

```diff
@@ -10,3 +10,43 @@ roleRef:
   kind: ClusterRole
   name: privileged-psp-user
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcd-backup
+rules:
+  - apiGroups:
+      - provider.giantswarm.io
+    resources:
+      - awsconfigs
+      - kvmconfigs
+      - azureconfigs
+    verbs:
+      - "get"
+      - "list"
+  - apiGroups:
+      - "extensions"
+    resources:
+      - ingresses
```

Comment:
i don't see where you get ingresses?

Reply:
right, i was using ingress before because it was easier, but then i realized we only have that in kvm